src/core/libmaven/mzSample.cpp

```diff
@@ -427,6 +414,36 @@ void mzSample::parseMzMLChromatogromList(xml_node &chromatogramList)
 	}
 }
 
+int mzSample::getSampleNoChromatogram(string chromatogramId) {
```

Comment:
const reference

Reply:
@rish9511 Does that matter? We are not passing pointer anyway.

Reply:
yes it does. 
passing parameters this way performs a copy operation which essentially creates a new object and then copies everything in that object. 

using references make sure that no new object is created.

Reply:
@rish9511 But object size if small here, right? Anyway, it is a good practice to use a const reference. Thanks for pointing out. I'll change here